spec/models/item_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Item, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
You did add a lot of Item functionality. Those class methods should be pretty easy to test since you know your API data so well.